tensorrt_llm/_torch/modules/fused_moe/fused_moe_cutlass.py

```
@@ -1,3 +1,18 @@
+# SPDX-FileCopyrightText: Copyright (c) 2026 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Ruff (0.15.18)

[warning] The file is executable but no shebang is present

(EXE002)